src/Resources/views/CRUD/list_array.html.twig

```diff
@@ -15,7 +15,6 @@ file that was distributed with this source code.
 {% block field %}
     {{ list.render_array(
         value,
-        field_description.options.inline is not defined or field_description.options.inline,
```

Comment:
If field_description.options.inline is not defined, the default value will be `true`.

With the current change, the default value will be `false`.

Reply:
This will be complicated, because on the other template it defaults to false

Reply:
We can either voluntary change the default behavior and update the doc about this

Or write something like 
```
{%set.options = { default_translation_domain : admin.translationDomain }|merge(field_description.options) %}

{% if options.inline is not defined %}
    options.inline = true
{% endif %}
```

Reply:
WDYT about doing like this?

```
{% block field %}
    {{ list.render_array(
        value,
        { default_translation_domain : admin.translationDomain, inline: true }|merge(field_description.options)
    ) }}
{% endblock %}
```

the other template with be default to false and this one to true

Reply:
Seems perfect

Reply:
Done